### ISSUE TITLE
Cannot enable static NAT on VM with NICs on multiple VPC tiers in UI

### ISSUE THREAD

**Issue by nvazquez** [apache/cloudstack#3334]
##### ISSUE TYPE
 * Bug Report

##### COMPONENT NAME
~~~
Network, VPC
~~~

##### CLOUDSTACK VERSION

~~~
4.11.2
~~~

##### CONFIGURATION
Advanced networking, VPC with multiple tier networks

##### OS / ENVIRONMENT
N/A

##### SUMMARY
When trying to enable static NAT on VM with NICs on multiple VPC tier networks on the 'Public IP addresses' router section, it fails with error: `VM ip <IP> address not belongs to the vm` where <IP> is the IP of the NIC tried to map for static NAT.

Note: The same process works under the tier network static NATs section, acquiring a new IP, but not from the VPC router section.

##### STEPS TO REPRODUCE

~~~
- Create VPC
- Add at least 2 tier networks
- Deploy VM on at least 2 of the VPC tier networks
- Network -> VPC -> Router -> Public IP addresses [1]
- Acquire new IP
- Enable static NAT: [2]
   - Select one of the tier networks
   - Select VM
   - Apply
~~~

[1] ![image](https://user-images.githubusercontent.com/5295080/58020060-73c28600-7add-11e9-91bc-5b4a322227ec.png)

[2] 
![image](https://user-images.githubusercontent.com/5295080/58020205-dddb2b00-7add-11e9-859f-c32d0188da9a.png)

##### EXPECTED RESULTS

~~~
Static NAT mapped correctly
~~~

##### ACTUAL RESULTS

~~~
VM ip <IP> address not belongs to the vm
~~~

**Comment by PaulAngus** [comment]
@nvazquez please can you detail where in the UI you are trying to add this IP.

**Comment by ustcweizhou** [comment]
patch

```
diff --git a/ui/scripts/vpc.js b/ui/scripts/vpc.js
index 8058f07..b7fac5b 100644
--- a/ui/scripts/vpc.js
+++ b/ui/scripts/vpc.js
@@ -39,7 +39,7 @@
                             url: createURL('listNics'),
                             data: {
                                 virtualmachineid: instance.id,
-                                nicId: instance.nic[0].id
+                                networkId: network.id
                             },
                             success: function(json) {
                                 var nic = json.listnicsresponse.nic[0];
```

**Comment by nvazquez** [comment]
Done @PaulAngus thanks

**Comment by ustcweizhou** [comment]
@nvazquez could you please test fix in #3338  ?

**Comment by nvazquez** [comment]
Thanks @ustcweizhou I'll test it
